test/pspm_extract_segments_test.m

```diff
@@ -157,8 +157,13 @@ function test_manual_length(this,nr_trial,nan_ratio)
       [control_data,timing] = generate_segment_data_manual(this, fn,nr_trial,nan_ratio);
       % do the actual test with options length all other option field
       % are set to default
-      [sts,out] = this.verifyWarningFree(@() ...
-        pspm_extract_segments('manual', fn,0, timing,struct('length',3.5)));
+      [sts,~] = this.verifyWarningFree(@() ...
+        pspm_extract_segments('file', fn, 0, timing,struct('length',3.5)));
+      this.verifyEqual(sts, 1);
+      % add testing 'screen' for nan_output
+      [sts,~] = this.verifyWarningFree(@() ...
+        pspm_extract_segments('file', fn, 0, timing,...
```

Comment:
I added this test because the `options.nan_output='screen'` is allowed according to the help text. However it does not work. This is related to the question left in `extract_segments`.

Reply:
fixed.